include/alpaka/kernel/TaskKernelGpuUniformedCudaHipRt.hpp

```diff
@@ -60,33 +70,34 @@ namespace alpaka
             namespace detail
             {
                 //-----------------------------------------------------------------------------
-                //! The GPU CUDA kernel entry point.
+                //! The GPU CUDA-HIP kernel entry point.
                 // \NOTE: 'A __global__ function or function template cannot have a trailing return type.'
                 template<
                     typename TDim,
                     typename TIdx,
                     typename TKernelFnObj,
                     typename... TArgs>
-                __global__ void cudaKernel(
+                __global__ void uniformedCudaHipKernel(
                     vec::Vec<TDim, TIdx> const threadElemExtent,
                     TKernelFnObj const kernelFnObj,
                     TArgs ... args)
                 {
 #if BOOST_ARCH_PTX && (BOOST_ARCH_PTX < BOOST_VERSION_NUMBER(2, 0, 0))
-    #error "Cuda device capability >= 2.0 is required!"
+    #error "UniformedCudaHip device capability >= 2.0 is required!"
```

Comment:
Can you please remove the `UniformedCudaHip` part here completely? `"Device capability >= 2.0 is required!"` should be enough:
```suggestion
    #error "Device capability >= 2.0 is required!"
```